lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -1000,4 +1000,14 @@ SPIRVToLLVMDbgTran::SplitFileName::SplitFileName(const string &FileName) {
   }
 }
 
+std::string SPIRVToLLVMDbgTran::findModuleProducer() {
```

Comment:
This function should be called in `SPIRVToLLVMDbgTran::transCompileUnit()`, right?

Reply:
Indeed, I do not know why it disappeared, I fixed it